app/javascript/react/screens/App/Plan/PlanReducer.js

```diff
@@ -57,6 +67,20 @@ const processVMTasks = vmTasks => {
       options: {}
     };
 
+    const hasPlaybookService = task.options.playbooks;
+
+    if (hasPlaybookService) {
```

Comment:
super minor nit, but we don't use `hasPlaybookService` anywhere but here, why not save a line, just do ` task.options.playbooks` here?